go.mod

```diff
@@ -10,6 +10,8 @@ replace github.com/denisenkom/go-mssqldb => github.com/denisenkom/go-mssqldb v0.
 // It's also present on grafana/loki's go.mod so we'll need till it gets updated.
 replace k8s.io/client-go => k8s.io/client-go v0.18.8
 
+replace github.com/grafana/grafana-aws-sdk => ../more/grafana-aws-sdk
```

Comment:
this is referencing the WIP/Draft code in:
https://github.com/grafana/grafana-aws-sdk/pull/8

Reply:
:+1: